include/View.hpp

```diff
@@ -46,7 +47,7 @@ class View : public ViewListeners
   Server *server;
   struct wlr_xdg_surface_v6 *xdg_surface;
   bool mapped;
-  int x, y;
+  FixedPoint<-4, int> x, y;
```

Comment:
What does all the "-4" means ? Is it like the offset for the window to be placed properly ? (aka the window borders)

Reply:
Otherwise everything looks good to me at the first sight

Reply:
-4 means 2^-4 precision, aka 1/16th.